packages/website/components/LiveEditor/ComponentSource.tsx

```diff
@@ -118,51 +147,78 @@ export function ComponentSource({
+        theme={github}
         // The order is important here
         scope={liveProviderScope}
       >
         <Card className={styles.card}>
           <LivePreview className={styles.previewWrapper} />
         </Card>
-        {showSource && (
-          <>
-            <LiveError className={styles.error} />
-            <div style={{ position: 'relative' }}>
-              <Flex className={styles.floatingControls}>
-                <CopyButton
-                  tooltipProps={{ placement: 'left' }}
-                  className={styles.copyButton}
-                  value={children}
-                />
-                {isExampleFromFile && (
-                  <Button
-                    className={styles.playgroundButton}
-                    endIcon={<ExternalLinkIcon />}
-                    onClick={() => {
-                      const href = `/playground?code=${coder.encode(children)}`;
-                      router.push(href, href);
-                    }}
-                  >
-                    Open in Playground
-                  </Button>
-                )}
-              </Flex>
+        <div style={{ position: 'relative' }}>
+          <LiveError className={styles.error} />
+          <div style={{ position: 'relative' }}>
+            <Flex
+              className={styles.toggle}
+              justifyContent="space-between"
+              alignItems="center"
+            >
+              <Button
+                size="small"
+                variant="secondary"
+                startIcon={
+                  showSource ? (
+                    <ComponentPreviewOffIcon />
+                  ) : (
+                    <ComponentPreviewIcon />
+                  )
+                }
+                onClick={handleToggle}
+              >
+                {showSource ? 'Hide code' : 'Show code'}
+              </Button>
+            </Flex>
 
-              <LiveEditor
-                style={{ paddingBottom: '45px', minHeight: '100px' }}
-              />
-            </div>
-          </>
-        )}
-        <Button
-          className={styles.toggle}
-          variant="secondary"
-          onClick={handleToggle}
-          isFullWidth
-        >
-          {showSource ? 'Hide source' : 'Show source'}
-        </Button>
+            <LiveEditor
+              className={cx(styles.editor, {
+                [styles.editorHidden]: !showSource,
+              })}
+            />
+            <Flex
+              className={styles.floatingPanel}
+              justifyContent="space-between"
+              alignItems="center"
+            >
+              {showSource && (
+                <Flex>
+                  <CopyButton
+                    tooltipProps={{ placement: 'top' }}
+                    className={styles.copyButton}
+                    value={children}
+                    size="small"
+                  />
+                  {isExampleFromFile && (
+                    <Flex marginLeft="spacing2Xs">
```

Comment:
we don't need this Flex I think

we can use `gap` in the Flex containing these buttons